owners/src/parser.js

```diff
@@ -47,10 +62,140 @@ class OwnersParser {
     this.logger = logger || console;
   }
 
+  /**
+   * Parse a single owner declaration.
+   *
+   * TODO(rcebulko): Add support for teams.
+   *
+   * @private
+   * @param {!string} ownersPath OWNERS.yaml file path (for error reporting).
+   * @param {!string} owner owner username.
+   * @return {OwnersParserResult<string[]>} list of owners' usernames.
+   */
+  _parseOwnersLine(ownersPath, owner) {
+    const owners = [];
+    const errors = [];
+
+    if (owner[0] === '@') {
```

Comment:
Is `owner` guaranteed to not be empty here?

Reply:
It's guaranteed by the method signature to be a string. It can (by signature) be an empty string (though it never will be because of how it's called), but that won't break this (`''[0] === undefined`)